src/executables/csql.c

```diff
@@ -117,6 +117,7 @@ int (*csql_text_console_to_utf8) (const char *, const int, char **, int *) = NUL
 
 int csql_Row_count;
 int csql_Num_failures;
+int csql_return_status;
```

Comment:
Could you verify that is used as global variable?

If then, I think one character of variable should be the upper character.
Like, csql_Row_count.(csql_row_count)

Reply:
Decided not to use global variable. 
Thank you.

Reply:
Ok, I got it!